pkgs/development/libraries/capstone/default.nix

```diff
@@ -7,15 +7,22 @@
 
 stdenv.mkDerivation rec {
   pname = "capstone";
-  version = "4.0.2";
+  version = "5.0-post1";
```

Comment:
Does it make sense to use 5.0-post1 if it has the same code as 5.0? Just a question.

Reply:
yes, because otherwise the update bot will create a PR with an update immediately